sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_parse_id.py

```diff
@@ -0,0 +1,28 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from ._shared import parse_key_vault_id, KeyVaultResourceId
+
+
+def parse_key_vault_key_id(source_id):
+    # type: (str) -> KeyVaultResourceId
+    """Parses a key's full ID into a class with parsed contents as attributes.
+    :param str source_id: the full original identifier of a key
```

Comment:
Needs a line break to render correctly:
```suggestion
    """Parses a key's full ID into a class with parsed contents as attributes.

    :param str source_id: the full original identifier of a key
```